app/scripts/reducers/dist.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+import assignDate from './assign-date';
+
+import {
+  DIST,
+  DIST_INFLIGHT,
+  DIST_ERROR
+} from '../actions/types';
+
+const initialState = {
+  dist: {
```

Comment:
Is there a reason to have a key of `dist` inside the redux store slice already named `dist`? It seems a bit repetitive.